CMakePresets.json

```diff
@@ -15,7 +15,10 @@
 				"lhs": "${hostSystemName}",
 				"rhs": "Windows"
 			},
-			"architecture": "Win32"
+			"architecture": "Win32",
+			"cacheVariables": {
+				"CMAKE_TOOLCHAIN_FILE": "C:/vcpkg/scripts/buildsystems/vcpkg.cmake"
+			}
```

Comment:
If vcpkg is installed somewhere else, will that preset work with manually defined CMAKE_TOOLCHAIN_FILE?